web/server/codechecker_server/session_manager.py

```diff
@@ -609,12 +649,14 @@ def create_session(self, auth_string):
             self.__cleanup_sessions()
 
         # Try authenticate user with personal access token.
-        auth_token = self.__try_auth_token(auth_string)
-        if auth_token:
-            local_session = self.__get_local_session_from_db(auth_token.token)
-            local_session.revalidate()
-            self.__sessions.append(local_session)
-            return local_session
+        if ":" in auth_string:
```

Comment:
What is the purpose of this condition? Could you give an example on the value of `auth_string` in case it doesn't come from username-password authentication?

Reply:
removed if , wasn't necessary anymore